src/assets/style/NativeShortcutGuide.css

```diff
@@ -27,6 +27,8 @@
 	--shortcut-guide-cell-shortcut-height: 1.5rem;
 	--shortcut-guide-cell-shortcut-width: 100px;
 	--shortcut-guide-explanation-font-size: 0.688rem;
+	--n90: #1c2950;
```

Comment:
We are not using this color naming anywhere else in the file. Can you standardize and call it something aligned with the other variables?